src/Microsoft.OData.Core/UriParser/Aggregation/ApplyClause.cs

```diff
@@ -102,6 +104,15 @@ internal List<string> GetLastAggregatedPropertyNames()
                     : CreateAggregatePropertiesUriSegment(aggregateExpressions);
             }
 
+            if (computeExpressions != null)
+            {
+                string computeProperties = string.Join(",", computeExpressions.Select(e => e.Alias));

```

Comment:
There's a build failure at this line:
`src\Microsoft.OData.Core\UriParser\Aggregation\ApplyClause.cs(109,61): error CS1503: Argument 2: cannot convert from 'System.Collections.Generic.IEnumerable<string>' to 'string[]' [src\Microsoft.OData.Core\Build.Net35\Microsoft.OData.Core.NetFX35.csproj]`

You may want to apply a `.ToArray()` at the end of the LINQ expression, similar to lines 93-94.